### PR TITLE
NOJIRA: Fixed issues in gradle build scripts

### PR DESCRIPTION
**Pull request by me2k**
This PR fixes issues in the maven publish step of the libraries as the source java classes were missing in the published files.

### CODE REVIEW

File: superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 /**
  * Created by gabriel.coman on 30/04/2018.
  */
-public class ResourceReader {
+public class ResourceReader {1
```
Comment (syhol) [inline superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java:9]: Whoops?

File: publish.gradle [hunk 2]
```diff
@@ -5,38 +5,56 @@ group = System.properties.getProperty("groupProj")
 def artifactProj = System.properties.getProperty("artifactProj")
 def gitUrl = 'https://github.com/SuperAwesomeLTD/sa-mobile-sdk-android.git'
 def libraryName = System.properties.getProperty("uploadName")
+def siteUrl = 'https://developers.superawesome.tv/extdocs/sa-mobile-sdk-android/html/index.html'
```
Comment (dnharjani-sa) [inline publish.gradle:8]: This should now be "https://sa-mobile-sdk-android.superawesome.tv/"

**Review by syhol — COMMENTED** [review]
Looks like theres 1 typo.

File: superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java (hunk 1 above)
Comment (dnharjani-sa) [inline superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java:9]: Errant 1?

File: superawesome-base/build.gradle [hunk 3]
```diff
@@ -3,10 +3,10 @@ apply plugin: 'kotlin-android'
 apply from: '../publish.gradle'
 
 ext.libraryDescription = 'SuperAwesome Base Mobile SDK for Android'
-
+ext.libraryName = 'SuperAwesome Base'
 android {
-    compileSdkVersion compile_sdk_version
-    buildToolsVersion build_tools_version
+    compileSdkVersion 29
+    buildToolsVersion '30.0.2'
```
Comment (dnharjani-sa) [inline superawesome-base/build.gradle:9]: We'll need to update the docs for api version if we're updating this

File: superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java (hunk 1 above)
Comment (me2k) [reply superawesome-base/src/test/java/tv/superawesome/lib/ResourceReader.java:9]: good spot!

File: superawesome-base/build.gradle (hunk 3 above)
Comment (me2k) [reply superawesome-base/build.gradle]: I've reverted this change

**Review by syhol — APPROVED** [review]

**Review by markGilchrist — APPROVED** [review]
I think the doc should update to the latest version

**Pull request merged**